### DISCUSSION TITLE
Archives and how do they get added?

### DISCUSSION THREAD

**Discussion by gary094** [chipzoller/hugo-clarity#188] [General]
Hello, thanks for your work on this beautiful theme!

I'm new to hugo but at this point I understand the basic structure.
However I cannot figure out how posts get added to the Archive.

By default post "Rich Content" is visible in the Archive but it seems it is not mentioned anywhere in the rich-content.md
I would assume there would be some kind of switch like "archive: true" (similar what draft: true does) but it is not. So I'm wondering how is it decided whether or not a post is marked as "archived"?

Or maybe is it intended to create a directory "archives" inside content and move any posts in there that I consider archived?

Also very strange to me is that the Archive in the header links to a directory "post/rich-content/" however there is no such directory in content directory. How is this generated then?

Thank you!

**Comment by onweru** [comment]
Hi @gary094 , `archives` is a simple menu label that links to the rich content page. It could have been any other text, and it could also link to any other page of your liking. This theme doesn't have an archives feature, yet. Since you're new to hugo, [this content organization video might help](https://www.youtube.com/watch?v=0GZxidrlaRM&list=PLLAZ4kZ9dFpOnyRlyS-liKL5ReHDcj4G3&index=6)

**Reply by zhugejun** [reply]
> Hi @gary094 , `archives` is a simple menu label that links to the rich content page. It could have been any other text, and it could also link to any other page of your liking. This theme doesn't have an archives feature, yet. Since you're new to hugo, [this content organization video might help](https://www.youtube.com/watch?v=0GZxidrlaRM&list=PLLAZ4kZ9dFpOnyRlyS-liKL5ReHDcj4G3&index=6)

Hi @onweru, I am having a hard time making the archives page work as an archives feature that lists all the posts grouped by year. How can I do it? Thanks.

**Reply by smurphboy** [reply]
I used this guide - https://cpbotha.net/2021/01/10/super-simple-yearly-post-archives-hugo/ - to add a by year archive to my blog. You can check it out at https://m7smu.org.uk/archives

**Reply by smurphboy** [reply]
If anyone can work out how to add tags after the URL before I do, then I'd be grateful. I'm assuming I need to write out the .Params.tags for each entry and style it with the button style?

**Reply by ainy0293** [reply]
@onweru 
<code>This theme doesn't have an archives feature.</code>

So, are there plans to add archives feature?

Many thanks

**Reply by onweru** [reply]
@ainy0293, I think we are going to add this due to popular demand. My schedule is tight but I will try to add it as soon as I have a chance.

**Reply by yabqiu** [reply]
I like this, and applied it to my blog, thanks.
